### PR TITLE
Blog post for Contributor Celebration 2021

### PR DESCRIPTION
**Pull request by Debanitrkl**
This PR is regarding the blogpost for contributor celebration 2021, the draft could be found [here](https://hackmd.io/@XzECIhDGRqCtGVrLiJFzxQ/BkUtTYwFY/edit)

### CODE REVIEW

File: content/en/blog/2021/contributor-celebration.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+4. **krafternetes**
+    A series of "studio sessions" by contributors, to showcase the art forms they are working on be it through live or recorded sessions, followed by a discussion on the same. Interested folks could join the #krafternetes channel on kubernetes slack. 
+
+### Schedule
+The event is scheduled from 16th-19th December 2021.
+More Information about the schedule could be found [here](https://kcc2021.sched.com/)
+
+
+### See you all at the event 
+If you have any questions or concerns, please visit [here](https://www.kubernetes.dev/events/kcc2021/faq/). Also don't forget to [register if you're interested](https://forms.gle/oAppmLDggEEGx5tz5) as well as bookmark the 2021 Kubernetes Contributor Celebration landing page for updated details between now and the celebration: <https://k8s.dev/celebration>
```
Comment (jberkus) [inline content/en/blog/2021/contributor-celebration.md:51]:
```suggestion
If you have any questions or concerns, please visit [here](https://www.kubernetes.dev/events/kcc2021/faq/). Also don't forget to [register if you're interested](https://forms.gle/oAppmLDggEEGx5tz5) as well as bookmark the 2021 [Kubernetes Contributor Celebration landing page](https://k8s.dev/celebration) for updated details between now and the celebration.
```

File: content/en/blog/2021/contributor-celebration.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+     At the end of the 90 minutes we’ll be joined by an esteemed group of expert judges who will judge your hopefully completed dessert on the following criteria:
+
+    1. Execution: How well does it fit the “Kubernetes theme”
+    2. Appearance: How does it look ?
+    3. Taste: Obviously the judges can’t taste it, so they’ll need to be creative for this one.
+    
+    **We’re looking for cooks of all backgrounds and talents.** If this sounds like fun to you and you want to cook with us, please fill the [form](https://docs.google.com/forms/d/e/1FAIpQLScmvXEFsdy0NzQofp198ShzupRPc4M5yDZYfdEMYPR4qD7aCw/viewform)
+
+3. [**Kubernetes Contributor Awards**](https://www.kubernetes.dev/events/kcc2021/activities/#kubernetes-contributor-awards)
+     In this special event we honor and dedicate the hard work that the community has been working on. These peer awards are a tradition at the Kubernetes Contributor Summits so we are bringing them virtual, please join us to thank and support all the people who have worked hard to help us this year.
```
Comment (jberkus) [inline content/en/blog/2021/contributor-celebration.md:40]:
```suggestion
     In this special event we honor contributors who have made special efforts, or shown extraordinary ability, during the prior year. These peer awards are a tradition at the Kubernetes Contributor Summits, and this year we're having them online. Please join us to thank and support fellow contributors who have made our project excellent.
```

File: content/en/blog/2021/contributor-celebration.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+ "DevOps Party Games takes the idea of "online party games" and tilts it on its head by adding DevOps-inspired content to existing games, and then streams it live via Twitch for a worldwide audience to watch, comment, and hopefully be entertained. In addition, the hosts (Matt Stratton, Jeremy Meiss, and Dan Maher) will provide color commentary, much like a modern day Cotton McKnight and Pepper Brooks (announcers from Dodgeball)." There will be a Kubernetes slant for this edition of DevOps Party Games.
+
+2. **[The Great Cloud Native Bakeoff](https://www.youtube.com/watch?v=koT08purWDc)**
+    A virtual cooking event styled after the [Great British Bake-off](https://en.wikipedia.org/wiki/The_Great_British_Bake_Off) will be held during the Kubernetes celebration Event 
+
+    Contestants will have 90 minutes to cook a Kubernetes themed dessert. Anything goes, as long as the majority of preparation is done live.  The competition is open to all skill levels, from baking from a cake mix to grinding your own flour as long as you can theme it to Kubernetes.
```
Comment (jberkus) [inline content/en/blog/2021/contributor-celebration.md:26]:
```suggestion
    Contestants will have 90 minutes to bake something related to to your Cloud Native experience. Anything goes, as long as the majority of preparation is done live.  The competition is open to all skill levels, from baking from a cake mix to grinding your own flour as long as it's somehow Kubernetes-themed.
```

File: content/en/blog/2021/contributor-celebration.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+To register, please fill out the [Registration Form](https://docs.google.com/forms/d/e/1FAIpQLSdRIboKulEPf7jVgRH3iTVwS4PEk6k0htcOs1eSG2OBmkxEjg/viewform).
+More deatils on [how to join](https://www.kubernetes.dev/events/kcc2021/how-to-join/) the event 
+
+### Events and Activities
+1. [**DevOps Party Games**](https://www.kubernetes.dev/events/kcc2021/activities/#most-extreme-kubernetes-challenge---devops-party-game)
+ "DevOps Party Games takes the idea of "online party games" and tilts it on its head by adding DevOps-inspired content to existing games, and then streams it live via Twitch for a worldwide audience to watch, comment, and hopefully be entertained. In addition, the hosts (Matt Stratton, Jeremy Meiss, and Dan Maher) will provide color commentary, much like a modern day Cotton McKnight and Pepper Brooks (announcers from Dodgeball)." There will be a Kubernetes slant for this edition of DevOps Party Games.
+
+2. **[The Great Cloud Native Bakeoff](https://www.youtube.com/watch?v=koT08purWDc)**
+    A virtual cooking event styled after the [Great British Bake-off](https://en.wikipedia.org/wiki/The_Great_British_Bake_Off) will be held during the Kubernetes celebration Event
```
Comment (jberkus) [inline content/en/blog/2021/contributor-celebration.md:24]:
```suggestion
    A virtual cooking event styled after the [Great British Bake-off](https://en.wikipedia.org/wiki/The_Great_British_Bake_Off) will be held during the Kubernetes celebration Event.
```

File: content/en/blog/2021/contributor-celebration.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+2. **[The Great Cloud Native Bakeoff](https://www.youtube.com/watch?v=koT08purWDc)**
+    A virtual cooking event styled after the [Great British Bake-off](https://en.wikipedia.org/wiki/The_Great_British_Bake_Off) will be held during the Kubernetes celebration Event 
+
+    Contestants will have 90 minutes to cook a Kubernetes themed dessert. Anything goes, as long as the majority of preparation is done live.  The competition is open to all skill levels, from baking from a cake mix to grinding your own flour as long as you can theme it to Kubernetes.
+    Contestants will be responsible for their own recipes and acquiring their ingredients. If you need help with either of these, let us know! 
+    Contestants will be expected to join a video conference call 30 mins prior to kick-off, and should have a phone or ipad with a stand (or a patient friend ) to capture them preparing their dessert.
+    Our Host and Commentators will provide commentary on our contestants and will ask each contestant questions throughout the event.
```
Comment (jberkus) [inline content/en/blog/2021/contributor-celebration.md:29]: please cut these three lines, it's too much detail, and is covered better by the form header.

File: content/en/blog/2021/contributor-celebration.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+     At the end of the 90 minutes we’ll be joined by an esteemed group of expert judges who will judge your hopefully completed dessert on the following criteria:
+
+    1. Execution: How well does it fit the “Kubernetes theme”
+    2. Appearance: How does it look ?
+    3. Taste: Obviously the judges can’t taste it, so they’ll need to be creative for this one.
```
Comment (jberkus) [inline content/en/blog/2021/contributor-celebration.md:35]:
```suggestion
At the end of the contest, each baked good will be judged on ambition, execution, success, and implementation of a Kubernetes theme.  The winner will get a unique, handmade, baking-related prize.
```

**Review by jberkus — CHANGES_REQUESTED** [review]

File: content/en/blog/2021/contributor-celebration.md (hunk 1 above)
Comment (mrbobbytables) [reply content/en/blog/2021/contributor-celebration.md:51]:
```suggestion
If you have any questions or concerns, please see the [faq](/events/kcc2021/faq/). Also don't forget to [register if you're interested](https://forms.gle/oAppmLDggEEGx5tz5) as well as bookmark the 2021 [Kubernetes Contributor Celebration landing page](https://k8s.dev/celebration) for updated details between now and the celebration.
```

**Review by mrbobbytables — CHANGES_REQUESTED** [review]

File: content/en/blog/2021/contributor-celebration.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+
+     At the end of the contest, each baked good will be judged on ambition, execution, success, and implementation of a Kubernetes theme.  The winner will get a unique, handmade, baking-related prize.
+    
+    **We’re looking for cooks of all backgrounds and talents.** If this sounds like fun to you and you want to cook with us, please fill the [form](https://docs.google.com/forms/d/e/1FAIpQLScmvXEFsdy0NzQofp198ShzupRPc4M5yDZYfdEMYPR4qD7aCw/viewform)
+
+3. [**Kubernetes Contributor Awards**](https://www.kubernetes.dev/events/kcc2021/activities/#kubernetes-contributor-awards)
+     In this special event we honor contributors who have made special efforts, or shown extraordinary ability, during the prior year. These peer awards are a tradition at the Kubernetes Contributor Summits, and this year we're having them online. Please join us to thank and support fellow contributors who have made our project excellent.
+     
+4. **krafternetes**
```
Comment (mrbobbytables) [inline content/en/blog/2021/contributor-celebration.md:35]:
```suggestion
4. **[krafternetes](/events/kcc2021/activities/#krafternetes)**
```

File: content/en/blog/2021/contributor-celebration.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+     A virtual cooking event styled after the [Great British Bake-off](https://en.wikipedia.org/wiki/The_Great_British_Bake_Off) will be held during the Kubernetes Celebration Event.
+
+    Contestants will have 90 minutes to bake something related to to your Cloud Native experience. Anything goes, as long as the majority of preparation is done live.  The competition is open to all skill levels, from baking from a cake mix to grinding your own flour as long as it's somehow Kubernetes-themed.
+
+
+     At the end of the contest, each baked good will be judged on ambition, execution, success, and implementation of a Kubernetes theme.  The winner will get a unique, handmade, baking-related prize.
+    
+    **We’re looking for cooks of all backgrounds and talents.** If this sounds like fun to you and you want to cook with us, please fill the [form](https://docs.google.com/forms/d/e/1FAIpQLScmvXEFsdy0NzQofp198ShzupRPc4M5yDZYfdEMYPR4qD7aCw/viewform)
+
+3. [**Kubernetes Contributor Awards**](https://www.kubernetes.dev/events/kcc2021/activities/#kubernetes-contributor-awards)
```
Comment (mrbobbytables) [inline content/en/blog/2021/contributor-celebration.md:32]:
```suggestion
3. [**Kubernetes Contributor Awards**](/events/kcc2021/activities/#kubernetes-contributor-awards)
```

File: content/en/blog/2021/contributor-celebration.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+### Registration
+To register, please fill out the [Registration Form](https://docs.google.com/forms/d/e/1FAIpQLSdRIboKulEPf7jVgRH3iTVwS4PEk6k0htcOs1eSG2OBmkxEjg/viewform).
+More details on [how to join](/events/kcc2021/how-to-join/) the event.
+### Events and Activities
+1. [**DevOps Party Games**](https://www.kubernetes.dev/events/kcc2021/activities/#most-extreme-kubernetes-challenge---devops-party-game)
+ "DevOps Party Games takes the idea of "online party games" and tilts it on its head by adding DevOps-inspired content to existing games, and then streams it live via Twitch for a worldwide audience to watch, comment, and hopefully be entertained. In addition, the hosts (Matt Stratton, Jeremy Meiss, and Dan Maher) will provide color commentary, much like a modern day Cotton McKnight and Pepper Brooks (announcers from Dodgeball)." There will be a Kubernetes slant for this edition of DevOps Party Games.
+
+2. **[The Great Cloud Native Bakeoff](https://www.youtube.com/watch?v=koT08purWDc)**
```
Comment (mrbobbytables) [inline content/en/blog/2021/contributor-celebration.md:22]:
```suggestion
2. **[The Great Cloud Native Bakeoff](/events/kcc2021/activities/#krafternetes)**
```

File: content/en/blog/2021/contributor-celebration.md [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+     A virtual cooking event styled after the [Great British Bake-off](https://en.wikipedia.org/wiki/The_Great_British_Bake_Off) will be held during the Kubernetes Celebration Event.
+
+    Contestants will have 90 minutes to bake something related to to your Cloud Native experience. Anything goes, as long as the majority of preparation is done live.  The competition is open to all skill levels, from baking from a cake mix to grinding your own flour as long as it's somehow Kubernetes-themed.
+
+
+     At the end of the contest, each baked good will be judged on ambition, execution, success, and implementation of a Kubernetes theme.  The winner will get a unique, handmade, baking-related prize.
+    
+    **We’re looking for cooks of all backgrounds and talents.** If this sounds like fun to you and you want to cook with us, please fill the [form](https://docs.google.com/forms/d/e/1FAIpQLScmvXEFsdy0NzQofp198ShzupRPc4M5yDZYfdEMYPR4qD7aCw/viewform)
```
Comment (mrbobbytables) [inline content/en/blog/2021/contributor-celebration.md:30]:
```suggestion
    **We’re looking for cooks of all backgrounds and talents.** If this sounds like fun to you and you want to cook with us, please fill the [form](https://docs.google.com/forms/d/e/1FAIpQLScmvXEFsdy0NzQofp198ShzupRPc4M5yDZYfdEMYPR4qD7aCw/viewform).
```

File: content/en/blog/2021/contributor-celebration.md [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+
+3. [**Kubernetes Contributor Awards**](https://www.kubernetes.dev/events/kcc2021/activities/#kubernetes-contributor-awards)
+     In this special event we honor contributors who have made special efforts, or shown extraordinary ability, during the prior year. These peer awards are a tradition at the Kubernetes Contributor Summits, and this year we're having them online. Please join us to thank and support fellow contributors who have made our project excellent.
+     
+4. **krafternetes**
+    A series of "studio sessions" by contributors, to showcase the art forms they are working on be it through live or recorded sessions, followed by a discussion on the same. Interested folks could join the #krafternetes channel on kubernetes slack. 
+
+### Schedule
+The event is scheduled from 16th-19th December 2021.
+More Information about the schedule could be found [here](/events/kcc2021/schedule/)
```
Comment (mrbobbytables) [inline content/en/blog/2021/contributor-celebration.md:40]:
```suggestion
More Information about the schedule could be found [here](/events/kcc2021/schedule/).
```

**Review by mrbobbytables — COMMENTED** [review]
Mostly formatting changes from previous suggestion 👍

**Comment by Debanitrkl** [comment]
Thanks @mrbobbytables I have updated with the changes I guess it can be merged now

File: content/en/blog/2021/contributor-celebration.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Announcing the Kubernetes Contributor Celebration 2021"
+linkTitle: "Contributor Celebration 2021"
+date: 2021-12-06
```
Comment (mrbobbytables) [inline content/en/blog/2021/contributor-celebration.md:4]:
```suggestion
date: 2021-12-07
```

**Comment by mrbobbytables** [comment]
Thanks! 
/label tide/merge-method-squash
/approve
/hold

@jberkus can you do a double check for me?

**Pull request merged**